packages/bsky/src/db/migrations/20230309T045948368Z-init.ts

```diff
@@ -232,68 +210,38 @@ export async function up(db: Kysely<unknown>): Promise<void> {
     .addPrimaryKeyConstraint('subscription_pkey', ['service', 'method'])
     .execute()
 
-  // @TODO(bsky) didHandle
+  // actor
   await db.schema
-    .createTable('did_handle')
+    .createTable('actor')
     .addColumn('did', 'varchar', (col) => col.primaryKey())
     .addColumn('handle', 'varchar', (col) => col.unique())
-    .addColumn('actorType', 'varchar')
-    .addColumn('declarationCid', 'varchar')
-    .execute()
-  await db.schema
-    .createIndex('did_handle_handle_lower_idx')
-    .unique()
-    .on('did_handle')
-    .expression(sql`lower("handle")`) // @TODO needed?
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .addColumn('takedownId', 'integer') // @TODO intentionally missing index
     .execute()
   await db.schema // Supports user search
-    .createIndex(`${'did_handle'}_handle_tgrm_idx`)
-    .on('did_handle')
+    .createIndex(`actor_handle_tgrm_idx`)
+    .on('actor')
     .using('gist')
     .expression(sql`"handle" gist_trgm_ops`)
     .execute()
 
-  // @TODO(bsky) repoRoot
-  await db.schema
-    .createTable('repo_root')
-    .addColumn('did', 'varchar', (col) => col.primaryKey())
-    .addColumn('root', 'varchar', (col) => col.notNull())
-    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
-    .addColumn('takedownId', 'integer') // @TODO intentionally missing index
-    .execute()
-
-  // @TODO(bsky) record
+  //record
   await db.schema
     .createTable('record')
     .addColumn('uri', 'varchar', (col) => col.primaryKey())
     .addColumn('cid', 'varchar', (col) => col.notNull())
     .addColumn('did', 'varchar', (col) => col.notNull())
-    .addColumn('collection', 'varchar', (col) => col.notNull())
-    .addColumn('rkey', 'varchar', (col) => col.notNull())
-    .addColumn('takedownId', 'integer') // @TODO intentionally missing index
+    .addColumn('json', 'text', (col) => col.notNull())
     .addColumn('indexedAt', 'varchar', (col) => col.notNull())
-    .execute()
-
-  // @TODO(bsky) ipldBlock
-  await db.schema
-    .createTable('ipld_block')
-    .addColumn('cid', 'varchar', (col) => col.notNull())
-    .addColumn('creator', 'varchar', (col) => col.notNull())
-    .addColumn('size', 'integer', (col) => col.notNull())
-    .addColumn('content', sql`bytea`, (col) => col.notNull())
-    .addPrimaryKeyConstraint('ipld_block_with_creator_pkey', ['creator', 'cid'])
+    .addColumn('takedownId', 'integer') // @TODO intentionally missing index
```

Comment:
why intentionally missing index?

Reply:
Mostly because I wasn't sure how we were going to land on this, but wanted to keep the column around since it's already all wired-up in the views.  I have a better sense of it as of this week— I will add the indexes back.

Reply:
Just looked at this— I forgot!  I wrote index but I probably meant foreign key, as the `moderation_report` table doesn't exist yet (I didn't know when writing this code whether it would ever exist).  I will update the comment.